components/TripMapComponent.tsx

```diff
@@ -1,39 +1,46 @@
-import { LocationData } from "expo-location";
+//import { LocationData } from "expo-location";
 import React from "react";
 import { StyleSheet, Dimensions } from "react-native";
-import MapView, { Polyline, Region, UrlTile } from "react-native-maps";
+import MapView, { EventUserLocation, Polyline, Region, UrlTile } from "react-native-maps";
 
 
-interface MapComponentProp {
-    onRegionChangeComplete: (region: Region) => void;
-    routePath: LocationData[];
+interface TripMapComponentProps {
+    onSheepLocChangeComplete: (region: Region) => void;
```

Comment:
For fremtidig refactoring: kalle den onSheep**Location**ChangeComplete for consistency med den andre callbacken